src/windows-nano/1803/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM mcr.microsoft.com/powershell:6.2.0-nanoserver-1803
+
+ENTRYPOINT C:\Windows\System32\cmd.exe
+
+WORKDIR C:\\Setup
```

Comment:
This layer doesn't seem necessary.  It feels like it is adding layer bloat to the image.

Reply:
Not sure I understand what to do, can you look at the latest version and clarify?

Reply:
Addressed